VERSION

```diff
@@ -1 +1 @@
-0.1.4-snapshot
\ No newline at end of file
+0.2.0-snapshot
```

Comment:
This is a kindness. I think semver would allow 0.1.5 to be a breaking change since all bets are off for v0.*.

Reply:
True. But we agreed to bump with @markfisher and @scothis